dist/performance.js

```diff
@@ -171,24 +160,70 @@ function getGamingMetrics(rawDoc) {
                     styleObj['top'] == '0px' &&
                     styleObj['left'] == '0px' &&
                     styleObj['z-index'] == '999'){
-                returnObj['fidIframeOverlay'] = true;
+                returnObj['fidIframeOverlayStrict'] = true;
             }
         }
+
+        let cover90viewport = false;
+        doesElementCoverPercentageOfViewport(iframeElement, 90)
+        returnObj['fidIframeOverlaySoft'] = cover90viewport && styleObj['z-index'] === findHighestZ(rawDoc);
+
     });
 
     return returnObj;
 }
 
+// Source: https://stackoverflow.com/questions/57786082/determine-how-much-of-the-viewport-is-covered-by-element-intersectionobserver
+// percentage is a whole number (ex: 90, not .9)
+const doesElementCoverPercentageOfViewport = (element, percentage) => {
+
+    const observer = new IntersectionObserver(function (entries) {
+        entries.forEach(function (entry) {
+            const entryBCR = entry.target.getBoundingClientRect();
+            const percentOfViewport = ((entryBCR.width * entryBCR.height) * entry.intersectionRatio) / ((window.innerWidth * window.innerHeight) / 100);
+            if (percentOfViewport > percentage) {
+                cover90viewport = true;
```

Comment:
Yes this variable is in the outer scope and has 2 different instances, ouch.

Reply:
Does this write to global scope? I wouldn't expect the other instance to be writable from this scope.

Since we know the scroll position will always be 0, is it possible to avoid the need for IntersectionObserver by looking at `entryBCR`'s x and y values, then immediately returning the boolean?